Gems/SceneProcessing/Code/Source/Generation/Components/MeshOptimizer/MeshOptimizerComponent.cpp

```diff
@@ -349,7 +349,10 @@ namespace AZ::SceneGenerationComponents
                 }
 
                 const AZStd::string name =
-                    AZStd::string(graph.GetNodeName(nodeIndex).GetName(), graph.GetNodeName(nodeIndex).GetNameLength()).append(SceneAPI::Utilities::OptimizedMeshSuffix);
+                    AZStd::string(graph.GetNodeName(nodeIndex).GetName(), graph.GetNodeName(nodeIndex).GetNameLength())
```

Comment:
This pull request should bump a version up to force FBX files to reprocess.

Reply:
how?

Reply:
The easiest would be to bump this version up https://github.com/o3de/o3de/blob/development/Gems/SceneProcessing/Code/Source/SceneBuilder/SceneBuilderWorker.cpp#L82